web/src/ui_init.js

```diff
@@ -532,7 +532,7 @@ export function initialize_everything(state_data) {
     stream_create.initialize();
     stream_edit.initialize();
     user_group_edit.initialize();
-    stream_edit_subscribers.initialize();
+    stream_edit_subscribers.initialize(state_data.max_subs_for_notification);
```

Comment:
I think we can directly use `realm.max_subs_for_notification` like we do for fields like `max_topic_length`.